packages/sns/test/utils/testContext.ts

```diff
@@ -58,8 +58,6 @@ export async function registerDependencies(
       },
       {
         lifetime: Lifetime.SINGLETON,
-        asyncDispose: 'destroy',
-        asyncDisposePriority: 40,
```

Comment:
Mmm, really interesting and good to know! It might be causing flakiness problems in library clients, right?

Reply:
yeah, I would expect so. we should also do this change on autopilot and check if it also works fine there